app/assets/stylesheets/cafe_works.css.scss

```diff
@@ -0,0 +1,19 @@
+.cafe-date {
+  width: 100px;
```

Comment:
Properties should be ordered background, border, border-color, border-radius, color, float, height, margin, padding, text-align, width